src/Qir/Execution/QirCommandLineTool/Program.cs

```diff
@@ -44,7 +46,27 @@ private static Command CreateBuildCommand()
             {
                 Handler = CommandHandler.Create((BuildOptions settings) =>
                 {
-                    return QirTools.BuildFromQSharpDll(settings.QSharpDll, settings.LibraryDirectories, settings.IncludeDirectories, settings.ExecutablesDirectory);
+                    var thisModulePath = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
```

Comment:
I believe @cesarzc has suggested that the logic for using these included resources be moved to the Qir.Tools project.

Reply:
The logic has been moved.